strawberryfields/configuration.py

```diff
@@ -218,6 +220,49 @@ def parse_environment_variable(key, value):
 
     return value
 
+def store_account(authentication_token, filename="config.toml", location="user_config", **kwargs):
+    """Stores an account in a configuration file.
+
+    The configuration file can be created in the following locations:
+       - user configuration directory ("user_config")
+       - current working directory ("local")
```

Comment:
```suggestion
    The configuration file can be created in the following locations:
     
     - A global user configuration directory ("user_config")
     - The current working directory ("local")
     
     This global user configuration directory differs depending on the operating system:
     
     * On Linux: ``~/.config/strawberryfields``
     * On Windows: ``~C:\Users\USERNAME\AppData\Local\Xanadu\strawberryfields``
     * On MacOS: ``~/Library/Application Support/strawberryfields``
     
     By default, Strawberry Fields will load the configuration and account credentials from the global
     user configuration directory, no matter the working directory. However, if there exists a configuration
     file in the *local* working directory, this takes precedence. The ``"local"`` option is therefore useful
     for maintaining per-project configuration settings.
```

Reply:
Would be good to also include examples here, using the standard `**Examples:**` heading we use in other docstrings 🙂

Reply:
This reads really good, thank you so much! :blush: